### PR TITLE
Add viscosity units

### PR DESCRIPTION
**Pull request by Tirael**
Add viscosity units

### CODE REVIEW

File: Src/Scripts/UnitDefinitions/Viscosity.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+﻿{
+    "Name": "Viscosity",
+    "BaseUnit": "Centistokes",
```
Comment (anjdreas) [inline Src/Scripts/UnitDefinitions/Viscosity.json]: I'm not familiar with this unit, but Centistokes struck me as maybe not the correct base unit? It seems to be "centi" of Stokes? For all the other units, we've stuck with the defined SI base unit if any, or as close to as what made sense.

Excerpt from wikipedia:
The SI unit of kinematic viscosity is m2/s. The cgs physical unit for kinematic viscosity is the stokes (St), named after George Gabriel Stokes. It is sometimes expressed in terms of centistokes (cSt).

Comment (anjdreas) [inline Src/Scripts/UnitDefinitions/Viscosity.json]: If we go with Stokes (which I have no idea if is a good base unit or not), then we have the benefit of reusing the prefix templating such as "centi", "kilo", "mega" etc.
From skimming through the wiki, it seems there are different types of viscosity such as kinematic, dynamic etc. Should we distinguish between these? Such as DynamicViscosity, KinematicViscosity? Or can you convert between kinematic viscosity units and dynamic viscosity units? I honestly don't know.

Comment (Tirael) [inline Src/Scripts/UnitDefinitions/Viscosity.json]: Thanks.
I'll correct this and update pull request.

**Comment by anjdreas** [comment]
And just a heads up, you need to comment on the PR for me to get an email notification. I'm not notified of new commits to a PR.

**Comment by Tirael** [comment]
Fixed incorrect kinematic viscosity description, unittests.

**Comment by anjdreas** [comment]
Great, can you squash to a single commit and rebase onto latest develop?

**Comment by anjdreas** [comment]
Or two commits, if you care to do the generated code into a separate commit. I won't require it though.

**Comment by Tirael** [comment]
I can not make rebase and squash branch feature/add-viscosity-unit, which has already been published. I propose to close the current pull request and open a new one with two commits.

**Comment by anjdreas** [comment]
Sure you can. Just rebase your feature branch onto develop, squashing commits so you get just a few, meaningful commits, then force push to your fork's feature branch - the one the pull request is based on. The pull request is updated automatically.

A couple of links:
https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request
http://stackoverflow.com/questions/17182624/contributing-to-project-on-github-how-to-rebase-my-pull-request-on-top-of-mast

I'm just trying to keep a clean, readable log. Let me know if you need any assistance.

**Comment by Tirael** [comment]
Squash and rebase are completed.

**Comment by anjdreas** [comment]
Awesome.

**Comment by anjdreas** [comment]
Nuget 3.9.0 is out.

**Pull request merged**